### DISCUSSION TITLE
Article: Capture errors and store them in a Supabase table

### DISCUSSION THREAD

**Discussion by shorn** [supabase/supabase#729] [Show and tell]
Article I wrote that uses Supabase as the data sink for capturing client error events.

[http://kopi.cloud/blog/2021/sentry-supabase/](http://kopi.cloud/blog/2021/sentry-supabase/)

The Supabase part shows a concrete example of using a stored procedure to insert rows in a table in a different schema.  Also, concrete example of using a Supabase REST endpoint directly without using the client library.

Note that I'm not advocating that capturing client events is a good use-case for Supabase. It was just an idea I had that I wanted to try out.

**Comment by kiwicopple** [comment]
This is awesome @shorn - I added it as a "Blog post" in the examples page:

https://supabase.io/docs/guides/examples#community
